backend/src/models/role.js

```diff
@@ -0,0 +1,16 @@
+import db from '../database/db.js';
+import { Sequelize } from 'sequelize';
+
+function Role(database) {
+  const Model = db(database).define('role', {
```

Comment:
```suggestion
  const Model = db(database).define('roles', {
```

**NOTA:** utilizar prural para los nombres de tablas.

Reply:
Listo.